D/DuckDB/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder, Pkg
 
 name = "DuckDB"
-version = v"0.3.0"
+version = v"0.3.1"
```

Comment:
You aren't actually changing the version of the source?

Reply:
I noticed that too. ;)